Documentation/gettingstarted/k8s-install-download-release.rst

```diff
@@ -1,6 +1,12 @@
-First, make sure you have `installed Helm`_ version 3.0.0 or higher.
+.. note::
```

Comment:
Sorry, can you add this to the file?

```
.. only:: not (epub or latex or html)

    WARNING: You are looking at unreleased Cilium documentation.
    Please use the official rendered version released here:
    http://docs.cilium.io
```

Reply:
Do we do that in files that we include?

Reply:
If we do this, I would have thought it would show up twice on the pages that include this snippet, once at the top of the outer page, and once at the start of this snippet that's included from the outer page.

@errordeveloper should be easy enough to test out with `make render-docs`.

Reply:
Ah, this would mostly be see if someone views the docs on GitHub, I'll add it.